src/hotspot/share/prims/jvmtiExport.cpp

```diff
@@ -3139,6 +3139,13 @@ bool JvmtiSampledObjectAllocEventCollector::object_alloc_is_safe_to_sample() {
     return false;
   }
 
+  // If the current thread is attaching from native and its Java thread object
+  // is being allocated, things are not ready for allocation sampling.
```

Comment:
nit - typo: s/is being allocated/has not been allocated/

Reply:
Please see other comment.